typescript/packages/wallets/sui/src/types.ts

```diff
@@ -0,0 +1,25 @@
+import { type SuiClient } from "@mysten/sui.js/client";
+
+export type AwesomeChainResponse = unknown;
+
+export type SuiTransaction = {
```

Comment:
This should just have a transaction object being passed

Reply:
The TransactionBlock class has been renamed to Transaction, and most methods and properties that previously used transactionBlock have been updated to use transaction instead.